nifi-nar-bundles/nifi-splunk-bundle/nifi-splunk-processors/src/test/java/org/apache/nifi/processors/splunk/TestPutSplunk.java

```diff
@@ -317,84 +260,39 @@ public void testUnableToCreateConnectionShouldRouteToFailure() {
         runner.assertAllFlowFilesTransferred(PutSplunk.REL_FAILURE, 1);
     }
 
-    /**
-     * Extend PutSplunk to use a CapturingChannelSender.
-     */
-    private static class UnableToConnectPutSplunk extends PutSplunk {
-
-        @Override
-        protected ChannelSender createSender(String protocol, String host, int port, int timeout, int maxSendBufferSize, SSLContext sslContext) throws IOException {
-            throw new IOException("Unable to create connection");
-        }
+    private void createTestServer(final String address, final TransportProtocol protocol) {
```

Comment:
Is `address` always `LOCALHOST`?  If so, then the `address` parameter could be removed in this method and the other `createTestServer` methods.